configs/12.0/packages/libpfm/sources

```diff
@@ -20,8 +20,8 @@
 #
 
 ATSRC_PACKAGE_NAME="Libpfm"
-ATSRC_PACKAGE_VER=4.10.1
-ATSRC_PACKAGE_REV=37d4628e37ba
+ATSRC_PACKAGE_VER=4.11.1
```

Comment:
@mscastanho is this version really correct?

Reply:
@tuliom I though so. I'm a bit confused. Should we keep using 4.10.1 then instead of 4.11.1? From your previous comment I though we should be using 4.11.1, but not a more recent revision.

Reply:
@mscastanho Oh! That was a typo... I meant to say 4.10.1.
The whole point is: why are we modifying this stable code?

The general rule is:
Going from 4.10.0 to 4.10.1 usually means that you're getting only bugfixes.
Going from 4.10.1 to 4.11.1 may also mean you're getting changes that may be incompatible with software built for 4.10. Sometimes we do have good reasons to have this kind of update, but I can't see it now.

Reply:
@tuliom This PR was meant to fix the issues seem by a Cherbot bump (#1535) which was updating libpfm to revision `5a623727cf71` (well past v4.10.1). Later we even got another automated bump (#1742) which tried to update libpfm to revision `fa84c27b6057` (current head of master).

So was Cherbot wrongly bumping this package in the first place? If that is the case, then this PR might not be needed at all.

Reply:
> So was Cherbot wrongly bumping this package in the first place? If that is the case, then this PR might not be needed at all.

If libpfm is going too far for AT 12.0, may be it's time for Cherbot to stop watching libpfm.
As @mscastanho said, sadly this PR won't be needed if we take nothing more than libpfm 4.10.1 on AT 12.0.